### PR TITLE
Flutter sample [Paint app]

### PR DESCRIPTION
**Pull request by ayushnishad**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,13 +70,16 @@ If you appreciate the content 📖, support projects visibility, give 👍| ⭐|
   
   [<img src = "https://github.com/OmiWakode/awesome-dart-pad/blob/master/art/BreatheRotate.gif" width=240>](https://dartpad.dev/14e84dd83f14c9df47971761463bb474)
 
-
 - [Matching the Pictures Game](https://dartpad.dev/482dbd401031ab9f74b0fa3de705ad2f) by [Sanskar Tiwari](https://github.com/theindianappguy) (09/03/2020)
   
   [<img src="/art/matchit.gif" width=240>](https://dartpad.dev/482dbd401031ab9f74b0fa3de705ad2f)
   
+- [Paint](https://dartpad.dev/a26d64f9f6805cc09dfa0743b10ae8b6) by [Ayush Nishad](https://github.com/ayushnishad)(15/03/2020)
```
Comment (divyanshub024) [inline README.md]: You need to add the sample to end of the list.

**Review by divyanshub024 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ayushnishad) [reply README.md]: i have upload it previously but still i will add it to the end.

**Comment by ayushnishad** [comment]
done.

**Comment by divyanshub024** [comment]
There are some conflicts. Can you resolve them?

**Comment by ayushnishad** [comment]
sure.

**Comment by ayushnishad** [comment]
@divyanshub024  all done.

**Comment by ayushnishad** [comment]
@divyanshub024 any changes required in #29 ?

**Review by divyanshub024 — APPROVED** [review]
LGTM!! Thanks a lot for the sample.

**Pull request merged**